script.js

```diff
@@ -0,0 +1,59 @@
+var choice = ["Rock", "Paper", "Scissors"];  
+choice.forEach(Function1);
+function Function1(){
+var btn = document.createElement("BUTTON");   
+btn.innerHTML = "index";                                
+document.getElementById("div1").appendChild(btn);
+}
+var player = prompt("Let's GOoOOOoOo");
+if (! player) {
+    
+    document.alret("should chose one of these optiones  " );
+} else {
+    
+    document.write("<p> player choice:" + " " + player+ "</p>" );
+}
+function computerChoice (){
+var computer= choice[Math.floor(Math.random() * choice.length)];
+
+        // Display computer choice
+        document.write("<p>Computer choice:" + " " + computer+ "</p>");
```

Comment:
don't use `document.write()` instead use `document.innerHTML()`
you can read about the problem using `document.write()` [here](https://developer.mozilla.org/en-US/docs/Web/API/Document/write)